staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -1023,6 +1023,10 @@ func DefaultBuildHandlerChain(apiHandler http.Handler, c *Config) http.Handler {
 	handler = genericapifilters.WithImpersonation(handler, c.Authorization.Authorizer, c.Serializer)
 	handler = filterlatency.TrackStarted(handler, c.TracerProvider, "impersonation")
 
+	handler = filterlatency.TrackCompleted(handler)
```

Comment:
this block should be above the authorizer so the authorization for the user making the request runs first.  Then this handler can do a second authorization check for the ActAs